### PR TITLE
Better: Update Apple Authenticate message specifications RD-24033

### PR DESCRIPTION
**Pull request by kalilz4485**
Changed: current Auth message
* Changes success status to "success" instead of "authenticated"
* includes the version in the response

New: new Auth message
* adds distinction with v1
* includes new payload

### CODE REVIEW

File: docs/interactions/structured-messages/apple-auth.md [hunk 1]
```diff
@@ -30,18 +33,73 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
...
+            "access_token":"ba371cc8c511e0d9114b8e17777ebc4ad5e23811144341f0ca4726d67ff4b19e",
+            "status":"success",
+            "version": 1
+          }
+        }
+      }
+    }
+  ]
+}
+```
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/apple-auth.md:99]: Screen Shot 2022-12-21 at 17 22 44
Screen Shot 2022-12-21 at 17 22 49

File: docs/interactions/structured-messages/apple-auth.md [hunk 2]
```diff
@@ -77,9 +135,15 @@ The customer will receive a message including the body and a button that will op
...
+            "code":"e993c351c5c031262b1c",
+            "status":"success",
+            "version": 2
+          },
           "authenticate": {
-            "token":"ba371cc8c511e0d9114b8e17777ebc4ad5e23811144341f0ca4726d67ff4b19e",
-            "status":"authenticated"
+            "code":"e993c351c5c031262b1c",
+            "status":"success",
+            "version": 2
```
Comment (kalilz4485) [inline docs/interactions/structured-messages/apple-auth.md:146]: Screen Shot 2022-12-21 at 17 22 58
Screen Shot 2022-12-21 at 17 23 03

File: docs/interactions/structured-messages/apple-auth.md [hunk 3]
```diff
@@ -30,18 +33,73 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
 |-|-|-|
 | **`source_id`** | String | ID of the source. |
 | **`in_reply_to_id`** | String | ID of the message being replied to. |
-| **`body`** | String | The authenticate structured content body. |
+| **`body`** | String | Content of the authentication message that the customer will receive. Optional if you provided the "Default title" field with a value in your channel settings. |
 | **`structured_content`** | Object | Payload of the structured message. |
 | **Structured Content Settings** | | |
 | **`structured_content.type`** | String | Type of the structured message. Must be "authenticate". |
-| **`structured_content.response_body`** | String | **Optional**. Field to be the body of the user’s response. If absent, default is the structured message body. |
+| **`structured_content.response_body`** | String | **Optional**. Field to be the body of the user’s response. If absent, the default is the structured message body. |
```
[the remark this answers is not in the document]
Comment (ylecuyer) [reply docs/interactions/structured-messages/apple-auth.md:40]: looks like the image is broken

[the remark this answers is not in the document]
Comment (kalilz4485) [reply docs/interactions/structured-messages/apple-auth.md:40]: Screen Shot 2022-12-21 at 17 22 28

**Comment by ylecuyer** [comment]
LGTM

**Pull request merged**